types/wcwidth/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for wcwidth 1.0
+// Project: https://github.com/timoxley/wcwidth
+// Definitions by: Junyoung Choi <https://github.com/rokt33r>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function wcwidth(input: string): number;
+
+declare namespace wcwidth {}
```

Comment:
Please remove this namespace augmentation. It is likely more appropriate to use `import wcwidth = require("wcwidth")`.

Reply:
I see. So, importing cjs module by `import * as someModule from './someModule'` is not intended.
(If I delete it, `import * as wcwidth from 'wcwidth'` will throw compiler error.)

Then, I think I should fix this PR too.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18195/files#diff-64e95c57399d3c780b2c8c79af515f29R1